### PR TITLE
Use Azure Storage for Spinnaker template

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*Don't use aka.ms links for template
*Use Azure Storage for Spinnaker template

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@EricJizbaMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (erijiz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: spinnaker-vm-simple/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Azure Spinnaker
 
-<a href="https://aka.ms/azspindeployqs" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fspinnaker-vm-simple%2Fazuredeploy.json" target="_blank">
```
Comment (arroyc) [inline spinnaker-vm-simple/README.md]: this looks good

File: spinnaker-vm-simple/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Azure Spinnaker
 
-<a href="https://aka.ms/azspindeployqs" target="_blank">
```
Comment (arroyc) [inline spinnaker-vm-simple/README.md]: wondering why was it here? its not standard and I doubt if it will work

Comment (EricJizbaMSFT) [inline spinnaker-vm-simple/README.md]: Mm there was a phase where I was told "CHANGE EVERYTHING TO AKA.MS LINKS" before we knew we had telemetry. It did work, but now it's just unnecessary

**Pull request merged**